assignments/prototypes.js

```diff
@@ -16,13 +16,34 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+function GameObject(attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+  this.destroy = function() {
+    return `${this.name} was removed from the game.`;
+  };
+}
+
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(charAttrs) {
+  GameObject.call(this, charAttrs)
+  this.healthPoints = charAttrs.healthPoints;
+  this.takeDamage = function() {
+    return `${this.name} took damage.`;
+  }
+
+}
+
+CharacterStats.prototype.destroy = Object.create(GameObject.prototype);
```

Comment:
On the left side of the `=` you have the `destroy` property of `CharacterStats` but on the right side, you have the entire prototype of `GameObject`.  Is that what you're looking to do?